apps/crn-frontend/src/network/working-groups/__tests__/WorkingGroupOutput.test.tsx

```diff
@@ -94,8 +94,6 @@ describe('WorkingGroupOutput', () => {
     userEvent.click(screen.getByText('Abu-Remaileh, M 1'));
     userEvent.click(screen.getByRole('textbox', { name: /Labs/i }));
     userEvent.click(screen.getByText('Example 1 Lab'));
-    userEvent.click(screen.getByRole('textbox', { name: /Authors/i }));
-    userEvent.click(screen.getByText('Person A 3'));
```

Comment:
?

Reply:
I moved it to the mandatory fields as authors are mandatory